bitwarden_license/src/Commercial.Infrastructure.EntityFramework/SecretsManager/Repositories/ServiceAccountRepository.cs

```diff
@@ -54,6 +54,23 @@ public async Task<bool> UserHasWriteAccessToServiceAccount(Guid id, Guid userId)
         return await query.AnyAsync();
     }
 
+    public async Task<IEnumerable<Core.SecretsManager.Entities.ServiceAccount>> GetPotentialGranteesAsync(Guid organizationId, Guid userId, AccessClientType accessType)
```

Comment:
The method name is fairly non descriptive. I believe this method is identical to `GetManyByOrganizationIdAsync` except we check write access instead?

`GetManyByOrganizationIdWriteAccessAsync` maybe.

Reply:
Resolved in https://github.com/bitwarden/server/pull/2628/commits/cd92ea867e0c629cdebeccbb8a3d9499053cef04